parser-typechecker/src/Unison/Codebase/Editor/HandleCommand.hs

```diff
@@ -190,6 +191,7 @@ commandLine config awaitInput setBranchRef rt notifyUser notifyNumbered loadSour
     Execute ppe uf -> void $ evalUnisonFile ppe uf
     AppendToReflog reason old new -> Codebase.appendReflog codebase reason old new
     LoadReflog -> Codebase.getReflog codebase
+    CreateAuthorInfo t -> AuthorInfo.createAuthorInfo Parser.Intrinsic t
```

Comment:
`Parser.External` here I think: `Intrinsic` is the location used for builtin functions. The definitions being created here are just regular pure Unison definitions that user could in principle create themselves.

Reply:
@pchiusano Cool, I will update that before merging.